code/modules/admin/fun_verbs.dm

```diff
@@ -929,79 +929,6 @@
 	log_admin("[key_name(IF)] started being imaginary friend of [key_name(friend_owner)].")
 	message_admins("[ADMIN_TPMONTY(IF)] started being imaginary friend of [ADMIN_TPMONTY(friend_owner)].")
 
-/datum/admins/proc/force_dropship()
-	set category = "Admin.Fun"
-	set name = "Force Dropship"
-
-	if(!check_rights(R_FUN))
-		return
-
```

Comment:
???

Reply:
Dead verb. There is already a shuttle manipulation window somewhere.